content/machine-controller/guides/proxy/_index.en.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Running behind a proxy"
+date = 2024-05-31T07:00:00+02:00
++++
+
+If nodes only have access via a HTTP proxy, you can let the machine-controller configure all new
+nodes to use this proxy. For this the following flag must be set on the machine-controller side:
+
+```bash
+-node-http-proxy="http://192.168.1.1:3128"
```

Comment:
This flag is no-op actually 😬 It's now integrated into OSM

Reply:
So what do? Remove the entire page?